### DISCUSSION TITLE
Welcome to Firefly III discussions!

### DISCUSSION THREAD

**Discussion by JC5** [firefly-iii/firefly-iii#4303] [General]
Hi everybody,

I'd like to support all the questions and requests for help I get, but I just run out of time. So I've set up this new fancy beta version of the "discussions" feature. Basically a forum. I hope with your help, we can support everybody who has questions about Firefly III.

  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Configuration support, help, etc.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

So, welcome everybody! Feel free to comment below with an introduction of yourself!

**Comment by JanFeld** [comment]
Great idea! I'm looking forward to get new input and even better ways to resolve issues, think about new features and help spreading the word of firefly-iii!
